diff.txt

```diff
@@ -0,0 +1,879 @@
+
+
+blade.portlet.osgiapi-1.0.0.jar
+*******************************************************
+
+
+blade.resourcebundle-1.0.0.jar
+*******************************************************
+CHANGED    MANIFEST   <manifest>
+ REMOVED    HEADER     Provide-Capability:osgi.service;objectClass:List<String>="java.util.ResourceBundle"
+ ADDED      HEADER     Provide-Capability:osgi.service;objectClass:List<String>="java.util.ResourceBundle",liferay.resource.bundle;bundle.symbolic.name="blade.resourcebundle";resource.bundle.base.name="content.Language"
+
+
+blade.rest-1.0.0.jar
+*******************************************************
+
+
+blade.service.hook.user-1.0.0.jar
+*******************************************************
+
+
+blade.servicebuilder.api-1.0.0.jar
+*******************************************************
+
+
+blade.servicebuilder.svc-1.0.0.jar
+*******************************************************
+CHANGED    MANIFEST   <manifest>
+ CHANGED    HEADER     Import-Package
+  ADDED      CLAUSE     blade.servicebuilder.service.impl
+  ADDED      CLAUSE     blade.servicebuilder.service.persistence.impl
+  CHANGED    CLAUSE     com.liferay.asset.kernel.service
+   REMOVED    PARAMETER  version:[1.0,2)
+  CHANGED    CLAUSE     com.liferay.asset.kernel.service.persistence
+   REMOVED    PARAMETER  version:[1.0,2)
+  CHANGED    CLAUSE     com.liferay.counter.kernel.service
+   REMOVED    PARAMETER  version:[1.0,2)
+  CHANGED    CLAUSE     com.liferay.expando.kernel.model
+   REMOVED    PARAMETER  version:[1.0,2)
+  CHANGED    CLAUSE     com.liferay.expando.kernel.util
+   REMOVED    PARAMETER  version:[1.0,2)
+  CHANGED    CLAUSE     com.liferay.exportimport.kernel.lar
+   REMOVED    PARAMETER  version:[1.0,2)
+  CHANGED    CLAUSE     com.liferay.portal.kernel.bean
+   REMOVED    PARAMETER  version:[7.0,8)
+  CHANGED    CLAUSE     com.liferay.portal.kernel.configuration
+   REMOVED    PARAMETER  version:[6.2,7)
+  CHANGED    CLAUSE     com.liferay.portal.kernel.dao.db
+   REMOVED    PARAMETER  version:[7.0,8)
+  CHANGED    CLAUSE     com.liferay.portal.kernel.dao.jdbc
+   REMOVED    PARAMETER  version:[7.0,8)
```

Comment:
nevermind, this is correct.  The portal kernel bundle may be 2.0.0 but the individual packages are still at 7.0